File: ios/RNCodeBase/NativeModules/TestModule/TestModule.m
//
//  TestModule.m
//  RNCodeBase
//
//  Created by Lam Nguyen on 03/08/2021.
//
#import <Foundation/Foundation.h>
#import "RNCodeBase-Bridging-Header.h"

@interface RCT_EXTERN_MODULE(TestModule, NSObject)

RCT_EXTERN_METHOD(add: (int*)a
                  b: (int*)b
                  callback: (RCTResponseSenderBlock)callback)
@end

File: ios/RNCodeBase/NativeModules/TestModule/TestModule.swift
//
//  TestModule.swift
//  RNCodeBase
//
//  Created by Lam Nguyen on 03/08/2021.
//

import Foundation
import React

@objc(TestModule)
class TestModule: NSObject{
  
  @objc func add(_ a: Int, b: Int, callback: @escaping RCTResponseSenderBlock) {
    callback([[
      "ans": a+b
    ]])
  }
}
